packages/nestjs-email/src/email.service.ts

```diff
@@ -19,6 +19,9 @@ export class EmailService {
     try {
       await this.mailerService.sendMail(dto);
     } catch (e) {
+      if (!(e instanceof Error)) {
```

Comment:
this should look like:

```
if (e instance of Error === false) {
  e = new NotAnErrorException(e);
}
// then if falls through to logger like normal
```

Reply:
we have this issue:
![image](https://user-images.githubusercontent.com/9724035/172856143-948adcf4-9c82-471a-b1e1-41fdad00ec55.png)